inc/autoload/class-scrollable-adminmenu.php

```diff
@@ -0,0 +1,41 @@
+<?php
+add_action( 'init', array( 'Enqueue_scrollable_adminmenu_Style', 'init' ) );
+
+class Enqueue_scrollable_adminmenu_Style {
+
+	private $suffix;
+
+	/**
+	 * Init function to register all used hooks.
+	 *
+	 */
+	public function __construct() {
+		$this->suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_style' ) );
+
+	}
+
+	/**
+	 * Initialize the class.
+	 */
+	public static function init() {
+		$class = __CLASS__;
+		if ( empty( $GLOBALS[ $class ] ) ) {
+			$GLOBALS[ $class ] = new $class;
+		}
+	}
+
+
+	public function admin_enqueue_style() {
+		$handle = 'enhance-adminmenu';
+		
+		wp_register_style(
+			$handle . 'styles',
```

Comment:
`styles` or `_styles`?

Reply:
the result for this is `enhance-adminmenu-styles` your proposal will end in `enhance-adminmenu-_styles`

Reply:
Then we should use `-styles`. The handle is only without ´-` on the end - `$handle = 'enhance-adminmenu';`

Reply:
you'r right its my fault. 
i'v fixed it from `enhance-adminmenustyles-css` to `enhance-adminmenu-css`